tests/main.js

```diff
@@ -402,12 +402,12 @@ testBundlers(
     t.deepEqual(
       functions,
       [
-        { name: 'four', mainFile: 'four.js/four.js.js', runtime: 'js', extension: '.js' },
```

Comment:
I've updated this test because even though the listed functions are the same, the order in which they're listed has changed, which shouldn't matter for the purposes of `listFunctions()`.